src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMessageHandlerTest.java

```diff
@@ -47,56 +60,118 @@
 
     @BeforeEach
     void setUp() {
-        when(interchange.getInterchangeHeader()).thenReturn(new InterchangeHeader());
+        lenient().when(interchange.getInterchangeHeader()).thenReturn(
```

Comment:
`lenient()` is new to me! Good to know about.

My solution in this kind of scenario was usually to split out the tests that needed the stubbing from those that didn't with `@Nested` classes. Using lenient() seems to change the statement from "when this happens then do this" to "if this ever happens then do this". 👍🏼